taichi/python/export_lang.cpp

```diff
@@ -772,10 +763,13 @@ void export_lang(py::module &m) {
   m.def("get_external_tensor_shape_along_axis",
         Expr::make<ExternalTensorShapeAlongAxisExpression, const Expr &, int>);
 
-  m.def("create_kernel",
-        [&](std::string name, bool grad) -> Program::KernelProxy {
-          return get_current_program().kernel(name, grad);
-        });
+  m.def(
+      "create_kernel",
+      [&](const std::function<void()> &body, const std::string &name = "",
```

Comment:
nit: kernel_name was a required argument before, is there any reason that we want to make it optional here? ;)

Reply:
fixed